### PR TITLE
KOGITO-3765 Kogito core should not depend on drools-mvel

### PR DESCRIPTION
**Pull request by evacchi**
jbpm is bringing the dependency. We make it optional so that it won't break native compilation.
https://issues.redhat.com/browse/KOGITO-3765

---

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.md)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

**Comment by evacchi** [comment]
so it begins

> [ERROR]       "message": "You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit"

**Comment by evacchi** [comment]
jenkins please retest this

**Comment by evacchi** [comment]
this seems to be **breaking** native compilation for some reason https://gist.github.com/evacchi/4763e0313c66d9e7895ec56f866e1014 so let's refrain from merging for now

File: drools/kogito-pmml/pom.xml [hunk 1]
```diff
@@ -43,6 +43,12 @@
       <groupId>org.kie.kogito</groupId>
       <artifactId>kogito-pmml-dependencies</artifactId>
       <type>pom</type>
+      <exclusions>
+          <exclusion>
+              <groupId>org.drools</groupId>
+              <artifactId>drools-mvel</artifactId>
+          </exclusion>
+      </exclusions>
```
Comment (danielezonca) [inline drools/kogito-pmml/pom.xml:51]: This has been removed from pmml with this PR https://github.com/kiegroup/drools/pull/3217 so it should part of 7.46

Comment (evacchi) [reply drools/kogito-pmml/pom.xml:51]: ok thx

**Comment by evacchi** [comment]
jenkins please retest this

**Comment by evacchi** [comment]
outdated

**Pull request closed without merging**